backend/src/main/java/ai/verta/modeldb/collaborator/CollaboratorServiceImpl.java

```diff
@@ -13,6 +13,7 @@
 import ai.verta.modeldb.monitoring.RequestLatencyResource;
```

Comment:
this file needs to be removed since collborator apis moved to uac

Reply:
moved everything to here: https://github.com/VertaAI/verta-uacservice/pull/189
But proto repo still needs to be updated